### ISSUE TITLE
Bug on cvsection

### ISSUE THREAD

**Issue by Erwan-BR** [posquit0/Awesome-CV#473]
If one of the cvsection has a space in the first 3 caracters, the space is just deleted and the two words are attached.

**Comment by OJFord** [comment]
Duplicate of #471, I suppose I'll leave this open though to have an actual issue (vs discussion) tracking it.

PR very welcome.

**Comment by dcsim** [comment]
This is a result of spaces not being tokenized and thus removed. 

Use a `~` instead of a space or a `\ `, similar to how you use a `\&` instead of just `&`.

**Comment by OJFord** [comment]
That's true, I was responding above to the different issue alluded to in the title: that you can't even compile with a header <3ch. We should fix that, and I suppose ideally the colouring would stop at the first word in the case it's <3ch and followed by a space.

**Comment by Kamalveer-Singh** [comment]
The issue is still open?? Is it for practicing purposes
